DOCS-68 Add note about removing installation media for VM. (#1608)
freenas/freenas-docs: userguide/virtualmachines.rst

   |          |                    |                | paravirtualized network drivers.                                                              |
   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
   | 4        | MAC Address        | string         | Enter the desired MAC address to override the auto-generated                                  |
   |          |                    |                | randomized MAC address.                                                                       |
   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
   | 4        | Attach NIC         | drop-down menu | Select the physical interface to associate with the VM.                                       |
   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
   | 5        | Optional: Choose   | browse button  | Click |ui-browse| to select an installer ISO or image file on the %brand%                     |

>    |          | installation media |                | system. After the media has been installed, delete the media device. Go to                    |

Reviewer: I don't think including this in the table here is the way to do it. Instead, add it in the text at the end of installing the VM.

   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
   | 5        | Optional: Choose   | browse button  | Click |ui-browse| to select an installer ISO or image file on the %brand%                     |
   |          | installation media |                | system.                                                                                       |
   |          | image              |                |                                                                                               |
   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
   | 5        | Upload ISO         | checkbox and   | Set to upload an installer ISO or image file to the %brand% system.                           |
   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+

The final screen of the Wizard displays the chosen options for the new
Virtual Machine (VM) type. Click :guilabel:`SUBMIT` to create the VM or

> :guilabel:`BACK` to change any settings. After the media has been

Reviewer: Better!  Removing the media is a change of subject, so should be in a new paragraph:
```suggestion
:guilabel:`BACK` to change any settings.

After the VM has been installed, remove the install media
```

> installed, delete the media device. Go to image

Reviewer: ```suggestion
device. Go to
```

:menuselection:`Virtual Machines -->` |ui-options| :menuselection:`--> Devices`.
Remove the *CDROM* device by clicking
|ui-options| :menuselection:`--> Delete`.

> This prevents %brand% from trying to boot with the installation media

Reviewer: ```suggestion
This prevents the virtual machine from trying to boot with the installation media
```

> even after it has been installed.

Reviewer: ```suggestion
after it has already been installed.
```
